lib/stdio/direction.js

```diff
@@ -10,24 +10,15 @@ import {isWritableStream} from './type.js';
 // This allows us to know whether to pipe _into_ or _from_ the stream.
 // When `stdio[index]` is a single value, this guess is fairly straightforward.
 // However, when it is an array instead, we also need to make sure the different values are not incompatible with each other.
-export const addStreamDirection = stdioStream => Array.isArray(stdioStream)
-	? addStreamArrayDirection(stdioStream)
-	: addStreamSingleDirection(stdioStream);
```

Comment:
Users can pass either a single value or an array of values to the `std*` option. At the moment, the logic handles both cases, for example in this line of code.

The main thing this PR does is to normalize single values to arrays (with a single item). This enables simplifying logic like this one.